codesign.py

```diff
@@ -395,7 +395,8 @@ def run_and_return_output(command):
         command,
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE)
-    return proc.stdout.readlines() + proc.stderr.readlines()
+    byteList = proc.stdout.readlines() + proc.stderr.readlines()
+    return [x.decode('utf-8') for x in byteList]
```

Comment:
what if a unicode character is more than one byte long?

Reply:
Will passing `text=True` do what we want? https://docs.python.org/3/library/subprocess.html#popen-constructor

Reply:
sounds like it: https://docs.python.org/3/library/subprocess.html#frequently-used-arguments

Reply:
Thanks for the reply! that's true, I will incorporate this change in the next commit!